NetCasbin/CoreEnforcer.cs

```diff
@@ -16,108 +16,116 @@ namespace NetCasbin
     /// </summary>
     public class CoreEnforcer
     {
-        private IEffector eft;
+        private IEffector _effector;
```

Comment:
Like your coding style, you are using a _ for private member and lower case the first character for protected member right? You can also try dotnet-format to format the code.

Reply:
Yes. I will try it.